data/net/minecraft/advancements/critereon/FishingRodHookedTrigger.mapping

```diff
@@ -3,12 +3,19 @@ CLASS net/minecraft/advancements/critereon/FishingRodHookedTrigger
 		ARG 1 json
 		ARG 2 entityPredicate
 		ARG 3 conditionsParser
+	METHOD lambda$trigger$0 (Lnet/minecraft/world/item/ItemStack;Lnet/minecraft/world/level/storage/loot/LootContext;Ljava/util/Collection;Lnet/minecraft/advancements/critereon/FishingRodHookedTrigger$TriggerInstance;)Z
+		ARG 3 instance
 	METHOD trigger (Lnet/minecraft/server/level/ServerPlayer;Lnet/minecraft/world/item/ItemStack;Lnet/minecraft/world/entity/projectile/FishingHook;Ljava/util/Collection;)V
 		ARG 1 player
 		ARG 2 rod
 		ARG 3 entity
 		ARG 4 items
 	CLASS TriggerInstance
+		METHOD <init> (Lnet/minecraft/advancements/critereon/EntityPredicate$Composite;Lnet/minecraft/advancements/critereon/ItemPredicate;Lnet/minecraft/advancements/critereon/EntityPredicate$Composite;Lnet/minecraft/advancements/critereon/ItemPredicate;)V
+			ARG 1 player
+			ARG 2 rod
+			ARG 3 entity
+			ARG 4 item
 		METHOD fishedItem (Lnet/minecraft/advancements/critereon/ItemPredicate;Lnet/minecraft/advancements/critereon/EntityPredicate;Lnet/minecraft/advancements/critereon/ItemPredicate;)Lnet/minecraft/advancements/critereon/FishingRodHookedTrigger$TriggerInstance;
```

Comment:
`item` ->`itemPredicate` (in `fishedItem`)

Reply:
Same case as previous, mojang used `item` as the predicate name.